### ISSUE TITLE
@inherits is not added to it's own line

### ISSUE THREAD

**Issue by wjonesy** [deanhume/html-minifier#19]
If a view has @inherits, the minifier does not put it on it's own line. But this does work for @model.

**Comment by wjonesy** [comment]
This also does not work for @using statements.

**Comment by deanhume** [comment]
Hi @wjonesy 
Thanks for logging this. Do you have a snippet of the code so that I can add it to the tests?

Thanks
Dean

**Comment by wjonesy** [comment]
@deanhume a snippet of razor code? where it doesn't minify correctly? See below

```
@inherits Umbraco.Web.Mvc.UmbracoTemplatePage
@{
    Layout = "~/Views/Shared/_Layout.cshtml";
}
@Umbraco.RenderMacro("BreadCrumb")
<div class="container-90 margin-bottom-large">
    <div class="row">
        <div class="col-xxs-24 col-xs-24 text-center padding-top-bottom-large">
            <h1 class="no-margin">@Model.Content.Name</h1>
        </div>
    </div>
</div>
```

and 

```
@using ClientDependency.Core.Mvc
@{
    Layout = null;
}

<!DOCTYPE html>
<!--[if lt IE 7 ]> <html class="ie6" lang="en"> <![endif]-->
<!--[if IE 7 ]>    <html class="ie7" lang="en"> <![endif]-->
<!--[if IE 8 ]>    <html class="ie8" lang="en"> <![endif]-->
<!--[if IE 9 ]>    <html class="ie9" lang="en"> <![endif]-->
<!--[if (gt IE 9)|!(IE)]><!-->
<html class="" lang="en">
<!--<![endif]-->
<head>
<title>@pageTitle</title>
</head>
<body></body>
</html>
```

**Comment by deanhume** [comment]
Hi @wjonesy 

Thats great thanks (it helps with testing). I'm looking into it and will let you know when its sorted.

**Comment by wjonesy** [comment]
There is also a case where you can have one @inherits and multiple @using statements in a file.

**Comment by deanhume** [comment]
This issue has been fixed in the latest version (1.6) - https://github.com/deanhume/html-minifier/releases/tag/v1.6

Thanks for submitting!
